src/frontend/components/ToggledSidebar/ToggledSidebar.css

```diff
@@ -0,0 +1,27 @@
+.shrinked-sidebar {
+  background-color: var(--color-background);
+  margin-left: 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.shrinked-sidebar-link {
+  cursor: pointer;
+  padding: 0.875rem;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  aspect-ratio: 1 / 1;
```

Comment:
Why aspect ratio is mentioned everywhere?

Reply:
it is based on this - https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio